src/Stripe.net/Entities/Subscriptions/Subscription.cs

```diff
@@ -45,9 +45,24 @@ public class Subscription : StripeEntity<Subscription>, IHasId, IHasMetadata, IH
         [JsonConverter(typeof(DateTimeConverter))]
         public DateTime? CancelAt { get; set; }
 
+        /// <summary>
+        /// If the subscription has been canceled with the <c>AtPeriodEnd</c> flag
+        /// set to true, <c>CancelAtPeriodEnd</c> on the subscription will be true.
```

Comment:
same here

I won't do all of them, we might just be okay with this for now. Let's talk to @ob-stripe on Monday